provisioning/Build.ps1

```diff
@@ -49,15 +49,18 @@ try {
     Write-OutPut "Npm.ps1 FAILED"
     exit 1
 }
-#try {
-#    $build = Join-Path $originalBuildScriptPath "Build-Windows.ps1"
-#    Write-OutPut "Running windows script: $build"
-#    iwr "$windowsBootstrapURL/Build.ps1" -UseBasicParsing -OutFile $build
-#    Invoke-Expression $build -skipNpm
-#} catch {
-#    Write-OutPut "Build.ps1 FAILED"
-#    exit 1
-#}
 
 $npm = "npm" -f $env:SystemDrive
 Invoke-Command $npm "install" $mainDir
+
+try {
+    $tests = Join-Path $originalBuildScriptPath "Tests.ps1"
+    $fullPath = Join-Path $originalBuildScriptPath "../node_modules/gpii-windows/provisioning/"
+    $args = "-originalBuildScriptPath $fullPath"
+    Write-OutPut "Running windows script: $tests"
+    iwr "$windowsBootstrapURL/Tests.ps1" -UseBasicParsing -OutFile $tests
```

Comment:
Yes, as Ale suggested during our last arch meeting and towards the goal of making the difference between the different build stages of our artifacts. In this particular case, the build process of the tests should not be part of the main build process but of the tests' build process. In summary, these are just small steps towards improving our current build chain.